build-logic/publishing/src/main/kotlin/gradlebuild.kotlin-dsl-plugin-bundle.gradle.kts

```
@@ -100,8 +108,8 @@ publishing {
 }
 
 gradlePlugin {
-    website.set("https://github.com/gradle/kotlin-dsl")
-    vcsUrl.set("https://github.com/gradle/kotlin-dsl")
+    website.set("https://github.com/gradle/gradle/tree/master/subprojects/kotlin-dsl-plugins")
+    vcsUrl.set("https://github.com/gradle/gradle/tree/master/subprojects/kotlin-dsl-plugins")
```

:lipstick: This would still point to the head of the default branch if we even change its name
```suggestion
    website.set("https://github.com/gradle/gradle/tree/HEAD/subprojects/kotlin-dsl-plugins")
    vcsUrl.set("https://github.com/gradle/gradle/tree/HEAD/subprojects/kotlin-dsl-plugins")
```